quantization/image_classification/cpu/run.py

```diff
@@ -52,13 +52,17 @@ def main():
     dr = resnet50_data_reader.ResNet50DataReader(
         calibration_dataset_path, input_model_path
     )
+
```

Comment:
call the pre-precess?

Reply:
It would break the flow described in Readme.md if we call pre-processing here.

Reply:
I think it might be better to make the pre-processing front and center. In the example, highlighting it to be a separated step. If I call it in her, it is kinda hidden. The user must dig into python code to locate it. In current layout, this step is on the command line level instead of python code level. This might be more obvious to the user. What do you think?